tools/schema_tools/prep_opsim.py

```diff
@@ -101,67 +138,214 @@ def offsetHex():
             vertex_count += 1
     return offsets
 
-def add_dither(database, simname, overwrite=True):
-    ## Add the Krughoff-Jones dithering pattern to the opsim output
-    ## adds three columns (ditheredRA, ditheredDec, and vertex) and indexes
-    #connect to the database
+def offsetRandom(noffsets, inHex=True):
+    """ 
+    Returns translational random dither offsets in radians.
+    
+    Required input
+    --------------
+    * noffsets: int: number of offsets needed.
+    
+    Optional input
+    --------------
+    * inHex: bool: set to False if dont want to restrict the dithers to within
+                   the hexagon inscribing the FOV. Default: True
+                  
+    """
+    import numpy as np
+    from lsst.sims.maf.stackers import inHexagon
```

Comment:
v4 will have a dependency on `sims_utils` for m5 calculation, so anything that it might want to use that is generic should go there. I like the idea of stripping `sims_utils` of `boost` since that package doesn't use it for anything other than unit testing, which seems like a heavyweight dependency.